### DISCUSSION TITLE
Help with a graphing bug

### DISCUSSION THREAD

**Discussion by anjakefala** [saulpw/visidata#2106] [General]
Hi @midichef!

I could use help with a graph debugging. 

It is hard to pinpoint which of your y-axis PRs resulted in this change, because I think there were multiple, and I suspect each of them affected it a little. I think you have a more intuitive understanding of how the y-axis calculation currectly works, and I could use your insight.

This is the expected behaviour when you zoom-in and out on a graph with your mouse-wheel:

![correct](https://github.com/saulpw/visidata/assets/7489659/53a7f0a8-451d-4606-851d-b7c67e5cd6da)

And this is the develop behaviour:

![incorrect](https://github.com/saulpw/visidata/assets/7489659/3df773cf-184a-4a46-89f7-1c3628927594)

Basically, zooming in and then back out should take you back to your original starting point. Now, it goes slightly off until eventually the plot disappears off the screen.

Do you have any ideas?

**Comment by midichef** [comment]
Hi @anjakefala, thanks for catching this, I'll look into it, in the next week or so.

**Reply by anjakefala** [reply]
@midichef Awesome! Could you write a quick comment here, so we can assign you =) https://github.com/saulpw/visidata/issues/2111
